docs/specification/1.0.md

```diff
@@ -58,18 +58,18 @@ The discovery endpoint is always available at `{baseUrl}/cds-services`. For exam
 The response to the discovery endpoint is an object containing a list of CDS Services.
 
 Field | Description
------ | -----------
+----- | ---------
 `services` | *array*. An array of **CDS Services**
 
 Each CDS Service is described by the following attributes.
 
-Field | Description
------ | -----------
-`hook`| *string* or *url*. The hook this service should be invoked on. See [Hook Catalog](#hook-catalog)
-`title`| *string*.  The human-friendly name of this service
-<nobr>`description`</nobr>| *string*. The description of this service
-`id` | *string*. The {id} portion of the URL to this service which is available at<br />`{baseUrl}/cds-services/{id}`
-`prefetch` | *object*. An object containing key/value pairs of FHIR queries to data that this service would like the EHR prefetch and provide on<br />each service call. The key is a *string* that describes the type<br />of data being requested and the value is a *string* representing<br />the FHIR query.<br />(todo: link to prefetching documentation)
+Field | Priority | Description
+----- | ----- | ---------
+`hook`| REQUIRED | *string* or *url*. The hook this service should be invoked on. See [Hook Catalog](#hook-catalog)
```

Comment:
Rather than REQUIRED, RECOMMENDED, and OPTIONAL, I think we should communicate just whether a field is required or not (`yes` or `no`), similar to PR #109.

The following fields should be required (indicated by a `yes` value in the column cell): `hook`, `title`, `description`, `id`.

`prefetch` is an optional field (indicated by a `no` value in the column cell). I do not think the specification should state that this field is recommended. There are certainly CDS Services that do not need or want any prefetched data.